homeassistant/components/reolink/media_source.py

```diff
@@ -57,7 +56,8 @@ async def async_resolve_media(self, item: MediaSourceItem) -> PlayMedia:
         _, config_entry_id, channel_str, stream_res, filename = identifier
         channel = int(channel_str)
 
-        host = self.data[config_entry_id].host
+        data: dict[str, ReolinkData] = self.hass.data[DOMAIN]
+        host = data[config_entry_id].host
```

Comment:
```suggestion
        host = self.hass.data[DOMAIN][config_entry_id].host
```

Reply:
Not sure if we need to explicitly type `host: ReolinkHost = ....`

Reply:
It's nicer to add the type annotation at the top level before we're accessing anything lower down in hass.data.

Reply:
Alright, makes sence